app/uk/gov/hmrc/exports/controllers/NotificationsController.scala

```diff
@@ -128,6 +135,16 @@ class NotificationsController @Inject()(
   private def findByEori(eori: String)(implicit hc: HeaderCarrier): Future[Result] =
     notificationsRepository.findByEori(eori).map(res => Ok(Json.toJson(res)))
 
+  private def findByEoriAndConversationId(eori: String, conversationId: String)(
+    implicit hc: HeaderCarrier
+  ): Future[Result] =
+    notificationsRepository.getByEoriAndConversationId(eori, conversationId).map { res =>
```

Comment:
And to retrieve only specific notifications searching based on eori and conversationId